src/SourceCode.Clay/Number.cs

```diff
@@ -929,20 +1018,30 @@ int IConvertible.ToInt32(IFormatProvider provider)
             }
         }
 
-        long IConvertible.ToInt64(IFormatProvider provider)
+        public double ToInt64()
+            => _int64; // Same but faster result as calling ToInt64(CultureInfo.InvariantCulture)
+
+        public long ToInt64(IFormatProvider provider)
```

Comment:
Not hidden behind explicit interface since it's a lowest-common-denominator method